### PR TITLE
Some fixes working towards a sales demo

### PR DESCRIPTION
**Pull request by jdunkerley**
### Pull Request Description

- Fixed the `write` format dropdown.
- Added `Text_Input` widget to path in various places.
- ICONs and Widgets in various places.
- Secret name dropdown.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [ ] Unit tests have been written where possible.
  - [ ] If GUI codebase was changed, the GUI was tested when built using `./run ide build`.

### CODE REVIEW

File: distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP/Response_Body.enso [hunk 1]
```diff
@@ -213,8 +214,7 @@ can_decode type = Meta.meta type . methods . contains "read_stream"
 decode_format_selector : Widget
 decode_format_selector =
     all_types = [Auto_Detect] + (format_types.filter can_decode)
-    make_name type_obj = type_obj.to_text.replace "_Format" "" . replace "_" " "
-    Single_Choice display=Display.Always values=(all_types.map n->(Option (make_name n) (File_Format.constructor_code n)))
+    Single_Choice display=Display.Always values=(all_types.flat_map .get_dropdown_options)
```
Comment (radeusgd) [inline distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP/Response_Body.enso:217]: oops, thanks for fixing that

File: distribution/lib/Standard/Base/0.0.0-dev/src/Widget_Helpers.enso [hunk 2]
```diff
@@ -4,12 +4,20 @@ import project.Data.Time.Date.Date
...
 from project.Data.Boolean import Boolean, False, True
 from project.Metadata import make_single_choice
 from project.Metadata.Choice import Option
 
+## PRIVATE
+   Creates a Single_Choice Widget for text or secret value.
+make_text_secret_selector : Widget
+make_text_secret_selector =
+    fqn = Meta.get_qualified_type_name Enso_Secret
+    make_single_choice [['<Text Value>', '""'], ['<Secret Value>', fqn+".get"]]
```
Comment (radeusgd) [inline distribution/lib/Standard/Base/0.0.0-dev/src/Widget_Helpers.enso:19]: Cool!

~~I'm wondering if we should also add a dropdown for names of secrets available at current working directory in `Enso_Secret.get`? (although that is relatively costly without caching)~~

~~Possibly one for next iterations :)~~

> Edit: oh how did I miss that you already added that 😅

**Review by radeusgd — APPROVED** [review]
Looks all good!

If not too annoying, perhaps we could add screenshots for the components whose icons are being changed? Just looking at the changed doc strings doesn't really tell much about the new appearance.

File: distribution/lib/Standard/Base/0.0.0-dev/src/Data.enso [hunk 3]
```diff
@@ -334,5 +335,6 @@ post (uri:(URI | Text)) (body:Request_Body=Request_Body.Empty) (method:HTTP_Meth
 @uri Text_Input
 download : (URI | Text) -> Writable_File -> HTTP_Method -> Vector (Header | Pair Text Text) -> File ! Request_Error | HTTP_Error
 download (uri:(URI | Text)) file:Writable_File (method:HTTP_Method=HTTP_Method.Get) (headers:(Vector (Header | Pair Text Text))=[]) =
-    response = HTTP.fetch uri method headers
-    response.write file
+    Context.Output.if_enabled disabled_message="Downloading to a file is forbidden as the Output context is disabled." panic=False <|
+        response = HTTP.fetch uri method headers
+        response.write file
```
Comment (radeusgd) [inline distribution/lib/Standard/Base/0.0.0-dev/src/Data.enso:340]: Thanks

File: distribution/lib/Standard/Base/0.0.0-dev/src/Widget_Helpers.enso (hunk 2 above)
Comment (jdunkerley) [reply distribution/lib/Standard/Base/0.0.0-dev/src/Widget_Helpers.enso:19]: agree - the `Enso_Secret.get` gives the default list in this cut.

**Review by AdRiley — APPROVED** [review]

**Pull request merged**